Sources/SnowManLogic/guessManager.swift

```diff
@@ -0,0 +1,29 @@
+public class GuessManager {
+	public var totalIncorrectGuessesAllowed: Int
+	public var correctGuesses: [String] = []
+	public var incorrectGuesses: [String] = []
+	public var currentGuess: String = ""
+
+	public init (totalIncorrectGuessesAllowed: Int) {
+		self.totalIncorrectGuessesAllowed = totalIncorrectGuessesAllowed
+	}
+
+	public func findUnguessedLetters(letters: [String]) -> Set<String> {
+		let setLetters = Set(letters)
+		let setCorrectGuesses = Set(correctGuesses)
+		return setLetters.subtract(setCorrectGuesses)
+	}
+
+	public func calculateRemainingGuesses() -> Int {
```

Comment:
Same question about the modifier here: Does this method need to be `public`? Is it intended to be used outside of this class?